samples/interactive-tutorials/src/main/java/product/ImportProductsBigQueryTable.java

```diff
@@ -40,10 +40,12 @@ public static void main(String[] args) throws IOException, InterruptedException
     String branchName =
```

Comment:
Can we move the below comment to line number 39? Something like:

// The name of the branch to query for products.
// To check for error handling, replace the below variable with the invalid branch name.
// String branchName = "invalid_branch_name";
String branchName = String.format(...)

Reply:
Everywhere else in the tutorials we have error handling below the valid value, so that user could just uncomment those lines of code without having two variables with the same name or two different variables.

Reply:
Refactored as described.